### ISSUE TITLE
Update check incorrectly reports previous version as latest

### ISSUE THREAD

**Issue by nielsarensman** [alejandrocoria/MapFrontiers#19]
When this mod is loaded on my modded 1.18.2 server the following two lines are visible:
[25Sept2022 13:23:46.353] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [mapfrontiers] Starting version check at https://alejandrocoria.games/projects/MapFrontiers/update.json
[25Sept2022 13:23:46.632] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [mapfrontiers] Found status: OUTDATED Current: 1.18.2-2.3.4 Target: 2.3.3

This shouldn't be flagged as outdated as version 2.3.4 should be reported as AHEAD due to the target not being updated. I guess it is reading the 1.18.2 part before the dash.

**Comment by alejandrocoria** [comment]
Yes, it is a problem that I have with Gradle in the Forge versions. I didn't spend time fixing it because it's not too annoying and I know I can fix it easily when I restructure the mod to use MultiLoader Template, unifying the Forge and Fabric source.

Thanks for letting me know, I'm going to leave this open to remind myself.
